atomicapp/providers/lib/kubeshift/kubernetes.py

```diff
@@ -52,7 +54,20 @@ def __init__(self, config):
         self.api.test_connection(self.k8s_api)
 
         # Gather the resource names which will be used for the 'kind' API calls
-        self.k8s_api_resources = self.api.get_resources(self.k8s_api)
+        self.k8s_api_resources = {}
+        self.k8s_api_resources['v1'] = self.api.get_resources(self.k8s_api)
+
+        # Gather what API groups are available
+        self.k8s_apis = urljoin(url, "apis/")
+
+        # Gather the group names from which resource names will be derived
+        self.k8s_api_groups = self.api.get_groups(self.k8s_apis)
+
+        for (name, versions) in self.k8s_api_groups:
+            for version in versions:
+                api = "%s/%s" % (name, version)
+                url = urljoin(self.k8s_apis, api)
+                self.k8s_api_resources[api] = self.api.get_resources(url)
```

Comment:
Nothing wrong with this part, but in the future when we add the refactored OpenShift code (that uses KubeBase) we should add this above section to KubeBase as well since both OpenShift and Kubernetes share the similar end-points (v1beta, etc.).

Thanks for the contribution @khrisrichardson !